css/style.css

```diff
@@ -85,6 +85,16 @@ body {
   position: relative;
 }
 
+.border {
```

Comment:
I think that this should be a different name, something like a "content-divider"

Reply:
you're right, idk what I was thinking. Done.